lib/helpers/searchPageHelpers.js

```diff
@@ -14,6 +14,9 @@ if ( Meteor.isClient) {
 		// },
 		search: function() {
 			return ListingSearch.getCurrentQuery();
+		},
+		urlsearch: function(){
+			return Session.get('query').q;;
```

Comment:
Unreachable ';' after 'return'.<br>Unnecessary semicolon.<br>'Session' is not defined.